drupal-org.make

```diff
@@ -104,8 +94,10 @@ projects[recaptcha][version] = 1.10
 
 projects[rules][version] = 2.3
 
-projects[services][version] = 3.7
+projects[services][version] = 3.12
 
+
+projects[services_documentation][download][branch] = 7.x-1.x
```

Comment:
The d.o packager requires both a branch and revision as a given revision can exist in multiple branches.